packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +63,44 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of items to be displayed as help menu content.
+   */
+  helpItems: PropTypes.arrayOf(PropTypes.shape(
+    navElementShape,
+    {
+      children: PropTypes.arrayOf(PropTypes.shape(
+      navElementShape,
+        {
+          children: PropTypes.array,
```

Comment:
do we want to limit them to this many children? Could do this recursively
https://stackoverflow.com/questions/32063297/can-a-react-prop-type-be-defined-recursively